fastly/resource_fastly_service_v1.go

```diff
@@ -1385,6 +1385,26 @@ func resourceServiceV1() *schema.Resource {
 					},
 				},
 			},
+			"dictionary": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						// Required fields
+						"name": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "Unique name to refer to this Dictionary",
+						},
+						// Optional fields
+						"dictionary_id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "Generated dictionary id",
```

Comment:
Change `id` to ``ID`

Reply:
Files were searched and updated where appropriate.